src/panels/config/script/ha-script-editor.ts

```diff
@@ -193,6 +195,17 @@ export class HaScriptEditor extends LitElement {
                             </paper-input>`
                           : html``}
                       </div>
+                      <div class="card-actions">
+                        <mwc-button
+                          @click=${this._runScript}
+                          title="${this.hass.localize(
+                            "ui.panel.config.script.picker.activate_script"
+                          )}"
+                          ?disabled=${this._dirty}
```

Comment:
Why disable when dirty?

Reply:
Because then it will run the saved version, and not the one you're seeing.

Reply:
I guess I could combine it, and add a confirmation dialog.
"Save changes and execute?" Yes/No